### DISCUSSION TITLE
Option to disable encryption?

### DISCUSSION THREAD

**Discussion by XBeg9** [yggdrasil-network/yggdrasil-go#1001] [Q&A] [answered]
First of all, thanks for such open-source software! 

The question is, if my app already sends encrypted traffic, I don't see a specific need to have Yggdrasil encrypt it again over transit. Is it possible to disable the encryption? 

Thasnk

**Comment by neilalexander** [comment, marked as answer]
There is no way to disable encryption, nope. It’s designed that way deliberately.
